src/components/RightSideDrawer.vue

```diff
@@ -0,0 +1,176 @@
+<template>
```

Comment:
- [ ]   Looks like you are just using this for a template to get is styled for now, but the final product for component will just introduce the other components ....
```
<profile v-if='showProfile' /></profile>
<submit-learn-more v-if='show-submit-learn-more'></submit-learn-more>
```

Would recommend putting the `showProfile` and `submitLearnMore` components in `src\components\sideDrawers` folder.